Jucer2CMake/main.cpp

```diff
@@ -427,8 +438,21 @@ int main(int argc, char* argv[])
         out << "  " << getSetting(exporter, "EXTRA_PREPROCESSOR_DEFINITIONS", "extraDefs")
             << "\n"
             << "  " << getSetting(exporter, "EXTRA_COMPILER_FLAGS", "extraCompilerFlags")
-            << "\n"
-            << ")\n"
+            << "\n";
+
+        if (exporterType == "XCODE_MAC")
+        {
+          out << "  "
+              << escape(
+                   "\\", getSetting(exporter, "PREBUILD_SHELL_SCRIPT", "prebuildCommand"))
```

Comment:
Escaping back-slashes at that point is wrong, since it's too late (we are escaping back-slashes that we added to escape the double-quotes). It should be moved to the `getSetting()` function.